targets/TARGET_NUVOTON/TARGET_M2351/device/partition_M2351_mem.icf

```diff
@@ -0,0 +1,82 @@
+/* See partition_M2351_mem.h for documentation */
```

Comment:
please add license header  to all new files if not yet in, like this one

Reply:
@0xc0170 Added license header.